tools/ci-scripts/linux/build_cpp-common.zsh

```diff
@@ -0,0 +1,15 @@
+#!/bin/zsh
+
+set -eux
+
+if [ ! $BOOST ]; then echo "BOOST not specified"; exit 1; fi
+
+export BOOST_ROOT=/opt/boosts/boost_`echo $BOOST | tr . _`
```

Comment:
Should we set the default value of `$BOOST` to be `1_63_0` and move this to `build.zsh`, instead of having a default value for `$BOOST_ROOT`?

Reply:
✔️